pyomo/contrib/mindtpy/single_tree.py

```diff
@@ -309,9 +269,9 @@ def add_lazy_affine_cuts(self, mindtpy_solver, config, opt):
                 try:
                     mc_eqn = mc(constr.body)
                 except MCPP_Error as e:
+                    config.logger.error(e, exc_info=True)
                     config.logger.debug(
-                        'Skipping constraint %s due to MCPP error %s'
-                        % (constr.name, str(e))
+                        'Skipping constraint %s due to MCPP error' % (constr.name)
                     )
                     continue  # skip to the next constraint
                 # TODO: check if the value of ccSlope and cvSlope is not Nan or inf. If so, we skip this.
```

Comment:
Should we address this now in this PR or leave it for later?

Reply:
This has already been checked in the following code. We can remove the TODO.